api/ocp/scc.go

```diff
@@ -0,0 +1,62 @@
+package ocp
+
+import (
+	"fmt"
+
+	secv1 "github.com/openshift/api/security/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewSecurityContextConstraints exports a default configuration of the
+// Security Context Constraints to deploy ceph-csi-operator in an OCP cluster
+func NewSecurityContextConstraints(namespace string, name string) *secv1.SecurityContextConstraints {
```

Comment:
we need to scc.yaml in config folder which is required for upstream users and we can use it as template here and make it customizable with serviceaccount name and namespace.

Reply:
add scc.yaml as a template, initially followed that route. but importing it threw some errors, so pivoted to generating the yaml from go code. 
but there was some issue with me pushing the package, that resulted the issue. pivoted back to the template

Reply:
There are some improvement required, lets talk about it in today's standup